test/season_test.rb

```diff
@@ -0,0 +1,37 @@
+require_relative './test_helper'
+require './lib/season'
+require 'csv'
+
+class SeasonTest < Minitest::Test
+  def setup
+    @season = mock('Season')
+    @season.stubs(:klass).returns('Season')
+    @season.stubs(:season).returns('20122013')
+    @season.stubs(:game_id).returns('2012030233')
+    @season.stubs(:type).returns('Postseason')
+    @season.stubs(:date_time).returns('5/20/13')
+    @season.stubs(:home_team_id).returns(17)
+    @season.stubs(:away_team_id).returns(16)
+    @season.stubs(:away_goals).returns(1)
+    @season.stubs(:home_goals).returns(3)
+    @season.stubs(:venue).returns('Dignity Health Sports Park')
+    @season.stubs(:venue_link).returns('/api/v1/venues/null')
+  end
```

Comment:
Excellent use of stubs here. I think stubbing the names of the identifiers really hooks us up for test performance and allows us to bypass a lot of running of the csv files over and over.